ros/src/tl_detector/infer.py

```diff
@@ -192,15 +190,15 @@ def infer(self, image_wrap, annotate, desired_labels = None, resize = True, conf
 
     def infer_and_save(self, image_file, desired_labels = None, resize = True, confidence_cutoff = 0.6):
         print(image_file)
-        image_wrap = ImageWrap(cv2.imread("assets\\" + image_file), True)
+        image_wrap = ImageWrap(cv2.imread("assets" + os.sep + image_file), True)
```

Comment:
It's good, but there is already a function called `os.path.join(...)` in case you didn't know.
It will handle any OS.